clicache/src/Cache.cpp

```diff
@@ -181,7 +181,7 @@ namespace Apache
       generic<class TKey, class TValue>
       array<Client::IRegion<TKey, TValue>^>^ Cache::RootRegions( )
       {
-        apache::geode::client::VectorOfRegion vrr;
+        std::vector<std::shared_ptr<apache::geode::client::Region>> vrr;
```

Comment:
While you're in the file, it would be nice to have some consistency.  Can we switch to use the `native` namespace?